Resources/Textures/Objects/Misc/desk_bell.rsi/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/blob/master/icons/obj/bureaucracy.dmi",
```

Comment:
you need to link the file *at* the commit (copy permalink)
![image](https://user-images.githubusercontent.com/60196617/227784534-4e33416a-6655-4f13-b7f7-00c7d3add00f.png)